### PR TITLE
feat: Add Zoom In and Zoom Out functionality and hotkeys

### PR DESCRIPTION
**Pull request by DmitryEremenko**
## Changes

-  Add Zoom In and Zoom Out functionality and hotkeys

## How to test it

- Open app, in View Menu click on Zoom In or Zoom Out item or use HotKey CTRL+= or CTRL+-

## Screenshots

- 
image

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Review by devcatalin — CHANGES_REQUESTED** [review]

File: src/constants/hotkeys.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ const hotkeys = {
   SELECT_FOLDER: 'ctrl+o,command+o',
   REFRESH_FOLDER: 'ctrl+f5,command+f5',
   TOGGLE_LEFT_PANE: 'ctrl+b,command+b',
+  ZOOM_IN: 'ctrl+=',
```
[the remark this answers is not in the document]
Comment (DmitryEremenko) [reply src/constants/hotkeys.ts:8]: Plus available only with Shift, hotkey in chrome use = as button to zoom in but label is +

[the remark this answers is not in the document]
Comment (devcatalin) [reply src/constants/hotkeys.ts:8]: You're right! Then just add the corresponding ones with `command` key

File: src/constants/hotkeys.ts [hunk 2]
```diff
@@ -5,6 +5,8 @@ const hotkeys = {
   SELECT_FOLDER: 'ctrl+o,command+o',
   REFRESH_FOLDER: 'ctrl+f5,command+f5',
   TOGGLE_LEFT_PANE: 'ctrl+b,command+b',
+  ZOOM_IN: 'ctrl+=,command+=',
+  ZOOM_OUT: 'ctrl+-,command+',
```
Comment (olensmar) [inline src/constants/hotkeys.ts:9]: great - please add to /docs/hotkeys.md also!

Comment (DYeromenko) [reply src/constants/hotkeys.ts:9]: Done!

Comment (DmitryEremenko) [reply src/constants/hotkeys.ts:9]: Done!

**Review by olensmar — APPROVED** [review]

**Comment by olensmar** [comment]
great - thanks!

File: src/constants/hotkeys.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (DmitryEremenko) [reply src/constants/hotkeys.ts]: Done!

**Review by olensmar — APPROVED** [review]

File: src/constants/hotkeys.ts [hunk 3]
```diff
@@ -5,8 +5,8 @@ const hotkeys = {
   SELECT_FOLDER: 'ctrl+o,command+o',
   REFRESH_FOLDER: 'ctrl+f5,command+f5',
   TOGGLE_LEFT_PANE: 'ctrl+b,command+b',
-  ZOOM_IN: 'ctrl+=',
-  ZOOM_OUT: 'ctrl+-',
+  ZOOM_IN: 'ctrl+=,command+=',
+  ZOOM_OUT: 'ctrl+-,command+',
```
Comment (devcatalin) [inline src/constants/hotkeys.ts:9]: should this be `command+-` ?

Comment (DmitryEremenko) [reply src/constants/hotkeys.ts:9]: indeed, thanks, fixed!

**Review by devcatalin — APPROVED** [review]

**Pull request merged**